bin/get_fulltext.py

```diff
@@ -2,16 +2,63 @@
 
 """get_fulltext CLI command"""
 
+import select
+import click
 import libfulltext
-from libfulltext.config import Config
+import libfulltext.config
+
+# Settings for click
+CLICK_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+
+@click.command(context_settings=CLICK_SETTINGS)
+@click.option("-c", "--config", default=libfulltext.config.DEFAULT_CONFIG_PATH,
+              type=click.File(), help="Path to the configuration file to use.")
+@click.argument("prefixed_ids", nargs=-1, default=None)
+@click.option("-f", "--prefixed-id-file", default="-", type=click.File(),
+              help="File with list of prefixed document identifiers, one per line. "
+              "By default '-', i.e. stdin, is chosen.")
+def get_fulltext(config, prefixed_ids, prefixed_id_file):
+    """
+    Obtain the fulltext pdfs for a number of documents,
+    which are identified by so-called prefixed document identifiers.
+
+    Examples for prefixed document identifiers:
+        doi:10.1016/j.cortex.2015.10.021
+    """
+    # Args:
+    #     config:            path to configuration file (string)
+    #     prefixed_ids:      list of prefixed document identifiers (list of strings)
+    #     prefixed_id_file:  plain text file with prefixed document identifiers (stream)
+
+    # Raises:
+    #     SystemExit: incompatible inputs (identifiers from multiple inputs)
```

Comment:
Why don't we have this in the docstring?

Reply:
The problem is that click uses the docstring to generate the `-h` message. (Should've written that in the comment and not just the commit message/review)
So if this was part of the docstring, we would get these descriptions on the commandline. But we wanted to still keep the docs in the source code.

Reply:
Alright, makes sense! :)